packages/web/components/Container.tsx

```diff
@@ -58,6 +58,11 @@ export const FullPageContainer: React.FC<
     bgSize="cover"
     spacing={0}
     p={{ base: 4, md: 8, lg: 12 }}
+    sx={{
+      scrollSnapAlign: 'start',
```

Comment:
Thanks for sorting the issues out @dan13ram . Is there some way to remove the momentary delay of the scrollSnap? When using keyboard it is nice and smooth but when scrolling with the mouse it scrolls to half way down the section and then smooth scrolls the other half and makes it feel quite sluggish. Also, I noticed that when on the Build section, the `IntroHero` buttons keep popping into view (can see it near the end of my recording), which I thought I had fixed.

https://user-images.githubusercontent.com/780350/156560339-d894b12b-e123-47c6-8f51-a49604aef49f.mp4

Reply:
Thanks @luxumbra. Unfortunately it looks like we can't customize the speed of scroll snap :/ 
See: https://stackoverflow.com/questions/56362168/how-long-is-the-delay-for-css-scroll-snap

I was able to fix that issue with the buttons peeping 😅